core/trino-main/src/main/java/io/trino/testing/TestingAccessControlManager.java

```diff
@@ -739,13 +739,17 @@ public List<ViewExpression> getRowFilters(SecurityContext context, QualifiedObje
     }
 
     @Override
-    public Optional<ViewExpression> getColumnMask(SecurityContext context, QualifiedObjectName tableName, String column, Type type)
+    public Map<ColumnSchema, ViewExpression> getTableColumnMasks(SecurityContext context, QualifiedObjectName tableName, List<ColumnSchema> columns)
     {
-        ViewExpression mask = columnMasks.get(new ColumnMaskKey(context.getIdentity().getUser(), tableName, column));
-        if (mask != null) {
-            return Optional.of(mask);
-        }
-        return super.getColumnMask(context, tableName, column, type);
+        Map<ColumnSchema, ViewExpression> superResult = super.getTableColumnMasks(context, tableName, columns);
+        return columns.stream()
+                .map(column ->
+                    Optional.ofNullable(columnMasks.get(new ColumnMaskKey(context.getIdentity().getUser(), tableName, column.getName())))
+                            .or(() -> Optional.ofNullable(superResult.get(column)))
+                            .map(viewExpression -> Map.entry(column, viewExpression)))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
```

Comment:
Using `flatMap` and streaming the optional would let you remove the `.filter(Optional::isPresent).map(Optional::get)`